lms/djangoapps/instructor/views/api.py

```diff
@@ -2368,11 +2370,12 @@ def list_report_downloads(_request, course_id):
     """
     course_id = CourseKey.from_string(course_id)
     report_store = ReportStore.from_config(config_name='GRADES_DOWNLOAD')
+    report_name = _request.POST.get("report_name", None)
```

Comment:
Please remove the `_` from `_request` since that parameter is now being used. It was presumably only there to avoid pylint warnings about an unused parameter.

Reply:
Is this new `report_name` parameter documented anywhere? It should at least be in the docstring for this method.